src/lib/plugins/live-cell/live-cell.tpl.html

```diff
@@ -0,0 +1,21 @@
+<ng-container [ngSwitch]="type">
+    <div class="live-cell {{ resultClass }} {{ [resultClass, type].join('-') }}" *ngSwitchCase="'currency'">
```

Comment:
maybe we should add template variable and set css classes in the component?

Reply:
use ngClass if possible